### DISCUSSION TITLE
Snapshot Camera Filtering

### DISCUSSION THREAD

**Discussion by dersgniw** [mrlt8/docker-wyze-bridge#924] [Q&A] [answered]
I want snapshots only for specific cameras.  I don't see a way to do this, but I might be missing something.

Probably silly, but I don't want the extra bandwidth associated with getting snapshots from all of my cameras every minute or so.

**Comment by mrlt8** [comment]
Snapshots are taken from the existing stream, so it shouldn't take any additional bandwidth over what is already being streamed.  That said, there are a couple of options. You can disable the stream in the bridge or set the snapshot refresh to 0 (or use the query: `http://localhost:5000/?refresh=0`) and manually refresh/stream the cameras you want to keep an eye on.

**Reply by dersgniw** [reply]
I probably wasn't clear.  I'm using RTSP(120) to take a picture every 2 minutes.  This works even when no stream is being viewed.  

I assume a connection is created, the image is taken and the connection is closed.  Is that correct?

**Reply by mrlt8** [reply, marked as answer]
Sorry, there isn't any way to filter out cameras using the SNAPSHOT option at this time, but I'll see if this is something we might be able to add in the future.

In the meantime, you could setup a crontab or systemd timer to curl the snapshot endpoint to take a new snapshot for the desired cameras.

**Reply by dersgniw** [reply]
Thank you.  Amazing application.
